cse/CommentReader.py

```diff
@@ -56,8 +80,22 @@ def readAllData(self):
             "comments": comments
         }
 
+    
+    def __loadAuthors(self):
+        self.__authors = {}
+        self.__authorsFile.seek(0)
+        fileFormat = next(self.__authorsReader)
+
+        for row in self.__authorsReader: # todo: don't load each author name. instead load seek offsets for each author id (less memory consumption). With delta encoding if necessary
+            self.__authors[row[0]] = row[1]
+
+
+
 
     def __parseRow(self, row):
+        raise DeprecationWarning("deprecated and not longer supported")
+        if not self.__authors:
```

Comment:
where do you use the authors in this method, that they must be loaded?

Reply:
deprecated method

Reply:
This doesn't answer my question. Why introduce new code in a deprecated method? And why do it in a way i will not work correctly?